index.html

```diff
@@ -0,0 +1,371 @@
+                    <i class="bi bi-star"></i>
+                </span>
+            </div>
+            <p class="section4-card__description">Slate helps you see how many 
+                more days you need to work to 
+                reach your financial goal.
+            </p>
+            <div class="section4-card__footer">
+                <div class="section4-card__footer-img">
+                    <img src="/src/Ellipse 2.png" alt="">
+                </div>
+                <div class="section4-card__profile">
+                    <h5 class="section4-card__names">Regina Miles</h5>
+                    <h5 class="section4-card__profession">Designer</h5>
+                </div>
+            </div>
+        </div>
+        <!-- card1  -->
+        <div class="section4-card">
+            <div class="section4-card__header">
+                <span>
+                    <i class="bi bi-star-fill"></i>
+                    <i class="bi bi-star-fill"></i>
+                    <i class="bi bi-star-fill"></i>
+                    <i class="bi bi-star-fill"></i>
+                    <i class="bi bi-star"></i>
+                </span>
+            </div>
+            <p class="section4-card__description">Slate helps you see how many 
+                more days you need to work to 
+                reach your financial goal.
+            </p>
+            <div class="section4-card__footer">
+                <div class="section4-card__footer-img">
+                    <img src="/src/Ellipse 3.png" alt="">
+                </div>
+                <div class="section4-card__profile">
+                    <h5 class="section4-card__names">Regina Miles</h5>
+                    <h5 class="section4-card__profession">Designer</h5>
+                </div>
+            </div>
+        </div>
+        <!-- card3  -->
+        <div class="section4-card">
+            <div class="section4-card__header">
+                <span>
+                    <i class="bi bi-star-fill"></i>
+                    <i class="bi bi-star-fill"></i>
+                    <i class="bi bi-star-fill"></i>
+                    <i class="bi bi-star-fill"></i>
+                    <i class="bi bi-star"></i>
+                </span>
+            </div>
+            <p class="section4-card__description">Slate helps you see how many 
+                more days you need to work to 
+                reach your financial goal.
+            </p>
+            <div class="section4-card__footer">
+                <div class="section4-card__footer-img">
+                    <img src="/src/Ellipse 4.png" alt="">
+                </div>
+                <div class="section4-card__profile">
+                    <h5 class="section4-card__names">Regina Miles</h5>
+                    <h5 class="section4-card__profession">Designer</h5>
+                </div>
+            </div>
+        </div>
+    </div>
+   </section>
+   <!-- Team section  -->
+   <section>
+        <div class="section5__content">
```

Comment:
Here you are using a block called `section5` with an element called `content` but that block is not defined yet at all! You can create it first before using it. Also please, try to use semantic names for your classnames because `section5` is not semantic.